docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md

```diff
@@ -0,0 +1,144 @@
+1. **Started**. A collection task is created on the Hosted Collector.
+1. **Initialized**. The task configuration is completed in Sumo Logic.
+1. **Authenticated**. The Source is successfully authenticated with Druva.
+1. **Collecting**. The Source is actively collecting data from Druva.
+
+If the Source has any issues during any one of these states, it is placed in an **Error** state.
+
+When you delete the Source, it is placed in a **Stopping** state. When it has successfully stopped, it is deleted from your Hosted Collector.
+
+On the Collection page, the [Health](/docs/manage/health-events#collection-page) and Status for Sources is displayed. Use [Health Events](/docs/manage/health-events) to investigate issues with collection. <br/>![C2C error icon on collection page.png](/img/send-data/C2C-error-icon-on-collection-page.png)
+
+Hover your mouse over the status icon to view a tooltip with a count of the detected errors and warnings.<br/> ![hover c2c error.png](/img/send-data/hover-c2c-error.png)
+
+## Setup and configuration Source
+
+When you create a Druva Source, you add it to a Hosted Collector. Before creating the Source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector.md).
+
+**To configure a Druva Source**
+
+1. In Sumo Logic, select **Manage Data** > **Collection** > **Collection**.
+1. On the **Collectors** page, click **Add Source** next to a Hosted Collector.
+1. Select **Druva**. <br/><img src={useBaseUrl('img/send-data/druva-icon.png')} alt="druva-icon.png" width="200"/>
+1. **Name**. Enter a Name to display for the Source in the Sumo Logic web application.
+1. **Description**. (Optional)
+1. **Source Category**.(Optional) Enter any string to tag the output collected from the Source. Category metadata is stored in a searchable field called `_sourceCategory`.
+1. Fields. (Optional) Click **+Add** to add additional fields; each field needs a name (key) and value.
+   * ![green check circle.png](/img/reuse/green-check-circle.png) A green circle with a check mark is shown when the field exists and is enabled in the Fields table schema.
+   * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point is shown when the field doesn't exist, or is disabled, in the Fields table schema. In this case, an option to automatically add or enable the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo that does not exist in the Fields schema or is disabled it is ignored, known as dropped.
+1. **Base URL**. Choose the base url from the drop-down menu as per your requirements — [Data Governance Cloud](https://apis.druva.com) and [Data GovernanceGov Cloud](https://govcloudapis.druva.com).<br/><img src={useBaseUrl('img/send-data/base-url.png')} alt="base-url.png" width="400"/>
```

Comment:
Broken image. It looks like you accidentally put the images in the docs folder  (`/docs/send-data`). Please move them to `/static/img/send-data`.